ping.py

```diff
@@ -281,4 +292,4 @@ def verbose_ping(dest_addr, timeout=2, count=1):
     # verbose_ping("heise.de")
     # verbose_ping("google.com")
     # verbose_ping("a-test-url-taht-is-not-available.com")
-    verbose_ping("192.168.10.1")
+    verbose_ping(configuration.CONFIGURATION.stratux_address)
```

Comment:
Are we sure that `.stratux_address` will be loaded in the self test?

Reply:
Not sure - I guess I assumed that it would be the same address, and since that value was already being stored in config...  anything that you find that you don't like, I'm totally good with your judgement.